docs/_dg/UseCases.md

```diff
@@ -78,11 +78,11 @@ Extensions:
 
 #### Use Case 4: Create Tag
 
-Preconditions: Tag does not yet exist in FoodRem.
+Preconditions: [[ tag:Tag ]] does not yet exist in FoodRem.
 
-MSS:
+[[ MSS ]]:
```

Comment:
Like OS, this is an invalid key as the glossary entry name is "MSS (Main Success Scenario)". However, I'm going to change this to "Main Success Scenario (MSS)" in another PR, so let's fix it the right way.

```suggestion
[[ main-success-scenario-mss:MSS ]]:
```